opensearch-jobs.yml

```diff
@@ -401,4 +355,1451 @@ instance_groups:
    - name: services
   env:
     bosh:
-      swap_size: 0
\ No newline at end of file
+      swap_size: 0
+
+- name: ingestor_logsearch_september_1
```

Comment:
These all should be only in `opensearch-production.yml`, because we don't want them in dev or stage